From bf8eeb610de434dc3adbdf0f4e8da39c70171cca Mon Sep 17 00:00:00 2001
From: Christoph Reiter <reiter.christoph@gmail.com>
Date: Fri, 23 Jul 2021 08:55:03 +0200
Subject: [PATCH 111/N] winconsoleio: build _testconsole which is required
 for the tests

This fixes test_winconsoleio

Should be merged into "build winconsoleio"
---
 setup.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/setup.py b/setup.py
index 4e7a284..63f486c 100644
--- a/setup.py
+++ b/setup.py
@@ -999,6 +999,9 @@ class PyBuildExt(build_ext):
                            ['_xxtestfuzz/_xxtestfuzz.c',
                             '_xxtestfuzz/fuzzer.c']))
 
+        if MS_WINDOWS:
+            self.add(Extension('_testconsole', ['../PC/_testconsole.c']))
+
     def detect_readline_curses(self):
         # readline
         if not MS_WINDOWS:
-- 
2.32.0

